ui/src/components/DiffTextArea/index.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { string, func, number } from 'prop-types';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import TextField from '@material-ui/core/TextField';
+import { withStyles } from '@material-ui/core/styles';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import { ReactGhLikeDiff } from 'react-gh-like-diff';
+import './index.css';
+
+@withStyles(theme => {
+  const borderColor =
+    theme.palette.type === 'light'
+      ? fade(theme.palette.common.black, 0.23)
+      : fade(theme.palette.common.white, 0.23);
+
+  return {
+    tab: {
+      flexGrow: 1,
+      color: theme.palette.text.primary,
+    },
+    tabContent: {
+      marginTop: theme.spacing.unit,
+      padding: theme.spacing.unit,
+    },
+    diffContainer: {
+      borderColor,
+      borderWidth: 1,
+      borderRadius: theme.shape.borderRadius,
+      borderStyle: 'solid',
+    },
+  };
+})
```

Comment:
The diff viewer looks good on the light theme of the site but looks weird on the dark theme. The text color shouldn't change since the background colors (e.g., green & red) are not changing.


#### Light theme

![Screen Shot 2019-09-03 at 10 16 07 AM](https://user-images.githubusercontent.com/3766511/64181143-f9bbd000-ce33-11e9-8001-4d2f8ec92e01.png)


#### Dark theme

![Screen Shot 2019-09-03 at 10 16 14 AM](https://user-images.githubusercontent.com/3766511/64181159-fe808400-ce33-11e9-8848-4661a199f618.png)

Reply:
If we `import 'react-gh-like-diff/lib/diff2html.css'`, the diff view layout might seem strange, so I add `index.css` to change some colors and remain other styles